src/frontend/components/DashboardThumbnail/DashboardThumbnail.spec.tsx

```diff
@@ -0,0 +1,184 @@
+import { flushAllPromises } from '../../testSetup';
```

Comment:
You have a typo in your commit title: `s/mange/manage/`

Reply:
And in the message: `s/sevreal/several/`

Reply:
:ok: